backend/app.js

```diff
@@ -13,20 +11,28 @@ app.use(bodyParser.json());
 // not used here, but you can parse urlencoded data too
 app.use(bodyParser.urlencoded({ extended: false }));
 
-var corsOption = {
-  origin: true,
-  methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
-  credentials: true,
-  exposedHeaders: ['x-auth-token']
-};
-app.use(cors(corsOption));
+app.use((req, res, next) => {
+  // for CORS
+  res.setHeader("Access-Control-Allow-Origin", "*");
+  res.setHeader(
+    "Access-Control-Allow-Headers",
+    "Origin, X-Requested-with, Content-Type, Accept, Authorization"
+  );
+  // for allowed methods
+  // OPTIONS is set implicitely by angular to check if
+  // post request is valid
+  res.setHeader(
+    "Access-Control-Allow-Methods",
+    "GET, POST, PATCH, PUT, DELETE, OPTIONS"
+  );
+  next();
+});
 
 // express.static allows requests to acess all images
 // path is used to forward request to /backend/images
-app.use("/images", express.static(path.join(__dirname, "images")));
+app.use("/images", express.static(path.join("images")));

```

Comment:
The one removed was the correct format. __dirname gives the absolute path during runtime